app/components/back-office-app-services/applications-command-handler-functions/function-app.tf

```diff
@@ -23,7 +23,7 @@ module "applications_command_handler_functions" {
   app_settings = {
     ServiceBusConnection__fullyQualifiedNamespace = "${var.service_bus_namespace_name}.servicebus.windows.net"
     SERVICE_BUS_HOSTNAME                          = "${var.service_bus_namespace_name}.servicebus.windows.net"
-    SERVICE_BUS_TOPIC                             = var.servicebus_topic_deadline_submission_topic_id
+    SERVICE_BUS_TOPIC                             = var.servicebus_topic_deadline_submission_topic_name

```

Comment:
is the topic name a different value to the topic id?

Reply:
Yeah the topic ID is the full Azure resource path, whereas the name is just `deadline-submission-topic`.

Reply:
Cool! Makes sense